### ISSUE TITLE
Group conv

### ISSUE THREAD

**Issue by thudzj** [f-dangel/backpack#122]
Hi,

Thanks a lot for such promising work! As the group conv is more and more popular in the CV community, do you have a plan to support the derivatives of group conv?

Best wishes!

**Comment by f-dangel** [comment]
Hi, thanks for your request.

Just to clarify your question: By _group conv_, are you referring to full support for the `groups` argument of [`Conv2d`](https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html#torch.nn.Conv2d) in BackPACK extensions? Most extensions currently only support `groups=1` but having the derivatives support `groups` would be possible. 

I am also curious to hear what is your specific use case for such layers.

Best!

**Comment by thudzj** [comment]
Hi,

Thank you for the prompt reply! Actually, I am working on estimating the diagonal fisher for the MobileNetV2 architecture. Namely, I only need to have access to the summation of the individual squared gradients (i.e., SumGradSquared). Yet, there are tens of depth-wise convolutions (i.e., convolutions with groups equivalent to the number of the channels of the input)  in the net, thus `extend` the model and `with backpack(SumGradSquared()):` raise the `NotImplementedError("Groups greater than 1 are not supported yet")`.

Thanks anyway!

**Comment by 815961618** [comment]
Hi, @f-dangel 
I uninstall original backpack and run "pip install --user git+https://github.com/f-dangel/backpack.git@development#egg=backpack", but I get fellowing error:


Collecting backpack
  Cloning https://github.com/f-dangel/backpack.git (to revision development) to /tmp/pip-install-lkiqxuxo/backpac
k_b2411083faa647178ef537c1025f9839
  Running command git clone -q https://github.com/f-dangel/backpack.git /tmp/pip-install-lkiqxuxo/backpack_b24110
83faa647178ef537c1025f9839
  Running command git checkout -b development --track origin/development
  Switched to a new branch 'development'
  Branch 'development' set up to track remote branch 'development' from 'origin'.
  WARNING: Generating metadata for package backpack produced metadata for project name backpack-for-pytorch. Fix your #egg=backpack fragments.
WARNING: Discarding git+https://github.com/f-dangel/backpack.git@development#egg=backpack. Requested backpack-for
-pytorch from git+https://github.com/f-dangel/backpack.git@development#egg=backpack has inconsistent name: filename has 'backpack', but metadata has 'backpack-for-pytorch'
ERROR: Could not find a version that satisfies the requirement backpack (unavailable)
ERROR: No matching distribution found for backpack (unavailable)

**Comment by f-dangel** [comment]
Hi @815961618,
thanks for the ping. I can reproduce your error:
- `pip=20.2.4`: working: `pip install --user git+https://github.com/f-dangel/backpack.git@development#egg=backpack`
- `pip=21.0.1`: failing: `pip install --user git+https://github.com/f-dangel/backpack.git@development#egg=backpack`

With `pip=21.0.1`, use 
```
pip install --user git+https://github.com/f-dangel/backpack.git@development#egg=backpack-for-pytorch
```
and it should work

**Comment by 815961618** [comment]
@f-dangel thank you so much.
